packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -1237,6 +1237,16 @@ class RenderSelectionSpy extends RenderProxyBox
   final Set<VoidCallback> listeners = <VoidCallback>{};
   List<SelectionEvent> events = <SelectionEvent>[];
 
+  @override
+  Size get size => _size;
```

Comment:
The browser context menu wrapper uses a Stack so a size is needed for the child of SelectableArea.

Reply:
Maybe copy this into a comment?